xml/troubleshooting.xml

```diff
@@ -1963,9 +1963,10 @@ nameserver <replaceable>ipaddress_of_nameserver</replaceable></screen>
          </para>
          <para>
           If your network connection is handled via DHCP, enable DHCP to change
-          host name and name service information by selecting <guimenu>Change
-          Hostname via DHCP</guimenu> and <guimenu>Update Name Servers and
-          Search List via DHCP</guimenu> in the &yast; DNS and Hostname module.
+          host name and name service information by selecting <guimenu>Set
+          Hostname via DHCP</guimenu> (can be set globaly for any interface or
```

Comment:
```console
$ echo "it can be set globaly" | aspell -a -l en
@(#) International Ispell Version 3.1.20 (but really Aspell 0.60.6.1)
*
*
*
*
& globaly 5 14: globally, global, glibly, globule, globular
```